code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -1,13 +1,13 @@
 #define EGG_INCUBATION_TIME 120
 
 /mob/living/simple_animal/hostile/headcrab
-	name = "Headslug"
+	name = "headslug"
 	desc = "Absolutely not de-beaked or harmless. Keep away from corpses."
 	icon_state = "headcrab"
 	icon_living = "headcrab"
 	icon_dead = "headcrab_dead"
-	health = 20
-	maxHealth = 20
+	health = 50
+	maxHealth = 50
 	melee_damage_lower = 5
 	melee_damage_upper = 5
```

Comment:
Make it do more damage, so if you get really lucky and or are good at combat you can infect someone by biting them to death. Like, 10-15.